runtime/doc/autocmd.txt

```diff
@@ -466,6 +466,14 @@ CompleteDone			After Insert mode completion is done.  Either
 				CompleteDonePre if you need it.
 				|v:completed_item| gives the completed item.
 
+				Sets these |v:event| keys:
+				    reason	Reason for completion being
+						done. Can be one of:
+						"accepted"   - completion was
+						               accepted
```

Comment:
I think this should be more explicit: _how_ is a completion accepted? (by `<c-y>`, or are there other ways?)

Same for `aborted`.

Reply:
> I think this should be more explicit: _how_ is a completion accepted? (by `<c-y>`, or are there other ways?)
> 
> Same for `aborted`.

are the updated docs good?

Reply:
Alignment is still a big mess. Consider using bullet points instead:
```
- "accepted": completion...
- "aborted": ...
```

Reply:
better now?